### ISSUE TITLE
Handle CC messages for glide, delay and crush

### ISSUE THREAD

**Issue by ggVGc** [datomusic/duo-imxrt#170]
I just came across that we're ignoring CC input for these parameters, but there are comments implying we want to react to them. This is also the case in the original brains1 firmware.

Should we handle them?
https://github.com/datomusic/duo-imxrt/blob/2faccfc3f1fed9388f41718bd19ffd5a0355361d/firmware/MidiFunctions.h#L174

**Comment by ggVGc** [comment]
@DavidMenting Please comment.

**Comment by DavidMenting** [comment]
The DUO only sends CC's but should not respond to them. We want the physical interface elements to always reflect the state of the synth. We could implement a 'local off' Sysex command to turn off local control, for instance for production testing or advanced use. For now, we can leave these unimplemented, or better yet, remove them altogether.

**Comment by ggVGc** [comment]
Yeah, let's just remove these lines then. I'll make a small PR associated with this issue so we have a paper trail of the decision.
